### ISSUE TITLE
different outputs

### ISSUE THREAD

**Issue by helia-mohamadi** [GiovanniPasq/DA-Faster-RCNN#9]
Hello
I used the model for my own dataset but every time I run the code , I've got different APs Percent.
I want to ask you about this problem.

**Comment by GiovanniPasq** [comment]
how much is the difference in APs?  Fluctuations of a few percentage points are partially related to the architecture and also may be due to a high variance present within your dataset or learning rate

**Comment by helia-mohamadi** [comment]
Yes, my dataset source and target class are different
Also, to have more experiment , I changed the learning rate value to 0.00002 and also changed the number of iterations to 100.

**Comment by GiovanniPasq** [comment]
In the unsupervised domain adaptation problem, the classes in the source domain must be the same of the target domain. You can have less classes than the source but they cannot be different (the target is of course unlabelled). 
For what i read i think the learning rate is low and also the number of iterations. 
NB: iterarions are not epochs, if you want to train the model for 100 epochs you need to calculate the number of iterations as follow:

Example considering a dataset length = 25 #length -> number of images in your dataset
```
iterations necessary to traing for 1 epoch = len(datasets)/batch
so in your case if your dataset has 25 images and batch is set to 1 you need 25 iterations to achieve 1 epoch (so if you need to do 100 epochs you need to multiply 25 * 100 => 2500 iterations).
If the batch is set to 2 you need 13 iterations to achieve 1 epoch (so if you need to do 100 epochs you need to multiply 13 * 100 => 1300 iterations).
```

**Comment by helia-mohamadi** [comment]
Thank you,
I use this formula to train the model better.
